docs/about.html

```diff
@@ -0,0 +1,283 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="shortcut icon" href="./images/favicon.png" type="image/x-icon">
+    <link rel="stylesheet" href="output.css">
+    <script src="main.js" defer></script>
+    <script src="https://kit.fontawesome.com/d3c3e6e26d.js" crossorigin="anonymous" defer></script>
+    <title>Garden and Flower</title>
+  </head>
+  <body class="font-sans min-w-[375px] overflow-x-scroll">
+    <header id="home">
+      <!-- Top Header -->
+
+      <div class="hidden md:block">
+        <nav class="h-[35px] bg-zinc-800 text-neutral-300 md:px-[5%] lg:px-[15%] flex justify-end content-center flex-wrap">
+          <ul class="flex gap-5">
+            <li class=" hover:text-rose-700">
+              <a href="https://www.facebook.com">
+                <i class="fa-brands fa-facebook"></i>
+              </a>
+            </li>
+
+            <li class=" hover:text-rose-700">
+              <a href="https://www.twitter.com">
+                <i class="fa-brands fa-twitter"></i>
+              </a>
+            </li>
+
+            <li class=" hover:text-rose-700">
+              <a href="#english">English</a>
+            </li>
+
+            <li class=" hover:text-rose-700">
+              <a href="#my-page">My Page</a>
+            </li>
+
+            <li class=" hover:text-rose-700">
+              <a href="#logout">Logout</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <!-- Desktop Header -->
+
+      <div class="hidden md:px-[5%] lg:px-[15%] md:py-2 md:flex justify-between items-center gap-8">
+        <a href="#home">
+          <img src="./images/flower-logo1.png" alt="Logo" width="150px">
+        </a>
```

Comment:
- [ ] As mentioned by the previous reviewers the logo of your page should take to the home page as this is part of the project requirements. So please make sure to implement it.  👇
```suggestion
        <a href="./index.html">
          <img src="./images/flower-logo1.png" alt="Logo" width="150px">
        </a>
```
- This also applies to the logo found in `index.html`